orchestrator/db/sorting/subscription.py

```diff
@@ -0,0 +1,20 @@
+from orchestrator.db import SubscriptionTable
+from orchestrator.db.sorting.sorting import generic_sort
+
+VALID_SORT_KEY_LIST = [
+    "subscription_id",
+    "product_id",
+    "customer_id",
+    "name",
+    "description",
+    "customer_descriptions",
```

Comment:
Not sure how sorting this will work out with multiple descriptions.

Reply:
did a test, added:
```
from orchestrator.db.filters.subscription import VALID_FILTER_FUNCTIONS_BY_COLUMN
from orchestrator.db.filters.generic_filters import generic_values_in_column_filter

VALID_FILTER_FUNCTIONS_BY_COLUMN.update({
    "test_filter": generic_values_in_column_filter(SubscriptionTable.status)
})
```
to our specific version, which gives the `test_filter` as new filterable in subscriptions query.

Reply:
does make the `VALID_FILTER_FUNCTIONS_BY_COLUMN` to generic, so will update the naming to be specific for the resource.

Reply:
commented on the wrong review comment

Reply:
is customer_descriptions SURF specific?

Reply:
yea, like customer id, removed it.